tapa/task.py

```diff
@@ -629,7 +629,9 @@ def add_rs_pragmas_to_fsm(self) -> None:
             f"{x}={x}" for x in (HANDSHAKE_START, *HANDSHAKE_OUTPUT_PORTS)
         )
         scalar_regex_str = "|".join(
-            x.name for x in self.ports.values() if not x.cat.is_stream
```

Comment:
Why is `x.cat.is_stream` not true if `x.is_streams` is true? This is very confusing.

Reply:
Discussed offline, this is a bug that needs to be fixed separately. LGTM to merge this commit as-is as a workaround.

Reply:
... except that maybe add a TODO?